src/Stratis.Bitcoin/Configuration/NodeSettings.cs

```diff
@@ -214,11 +217,13 @@ public NodeSettings LoadConfiguration(List<IFeatureRegistration> features = null
                 this.ConfigurationFile = this.CreateDefaultConfigurationFile(features);
             }
 
+            // Add the file configuration to the command-line configuration.
             var fileConfig = new TextFileConfiguration(File.ReadAllText(this.ConfigurationFile));
             var config = new TextFileConfiguration(args);
             this.ConfigReader = config;
             fileConfig.MergeInto(config);
 
+            // TODO: Does this code belong here?
```

Comment:
Probably better if it's in the DBreezeCoinView ctor, you're right.
This way, if the coinView is not used, for example in Breeze, the folder is not created.
Also, the check `if (!Directory.Exists(this.DataFolder.CoinViewPath))` is redundant.

Reply:
Good idea. I've made the change.